task_DEV_5/task_DEV_5/GenerateFile.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class GeneratorFile
+    {
+        private const string INVALID_LINE_LENGTH_ERROR_MESSAGE = "String length cannot be negative.";       
+        private const int MAX_STRING_LENGTH = 101;
+        private const int MIN_STRING_LENGTH = 1;
+        private string path;
+        private int stringAmount;
+
+        public GeneratorFile(string path, int stringAmount)
+        {
+            this.path = path;
+            this.stringAmount = stringAmount;
+        }
+
+        public void CheckStringLength()
+        {
+            if (stringAmount < 0)
+            {
+                throw new ArgumentOutOfRangeException(INVALID_LINE_LENGTH_ERROR_MESSAGE);
+            }
+        } 
+
+        public void WriteToFile()
```

Comment:
This method generates new file, not just write smth. Better to name it `Generate`.

Reply:
Done.